app/commands/decidim/participations/admin/update_participation.rb

```diff
@@ -24,10 +26,13 @@ def call
 
           transaction do
             update_participation
+            send_notification_moderation if @participation.unmoderate?
```

Comment:
@ludivinecp why do we send it here instead of the end ?
We need to discuss that, as it seems in some method we check some object attributes that have not been updated yet

Reply:
We need to check if the participation has already been moderate or not. For that we have to check his moderation status before it'll be update.